File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.py
class Solution(object):
    def appendCharacters(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: int
        """
        s_ = len(s)
        t_ = len(t)
        i = 0
        j = 0
        while i < s_:
            if j < t_ and s[i] == t[j]:
                i+=1
                j+=1
            else:
                i+=1
        return t_ - j

File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.cpp
class Solution {
public:
    int appendCharacters(string s, string t) {
        int s_ = s.size();
        int t_ = t.size();
        int i = 0;
        int j = 0;
        while(i < s_){
            if (j < t_ & s[i] == t[j]){
                i++;
                j++;
            }
            else{
                i++;
            }
        }
        return t_ - j;
    }
};